spring-cloud-tencent-examples/polaris-ratelimit-example/ratelimit-callee-service/src/main/resources/bootstrap.yml

```diff
@@ -8,6 +8,9 @@ spring:
       address: grpc://127.0.0.1:8091
       namespace: default
       enabled: true
+      discovery:
```

Comment:
No need to register service instance in example of rate-limit.

Reply:
Here just want to use the service discovery capability, the instance does not need to be registered to the registry.

Reply:
> No need to register service instance in example of rate-limit.

Here just want to use the service discovery capability, the instance does not need to be registered to the registry.